x-pack/plugins/beats_management/public/pages/tag/index.tsx

```diff
@@ -96,6 +101,17 @@ export class TagPageComponent extends React.PureComponent<TagPageProps, TagPageS
       </PrimaryLayout>
     );
   }
+  private rgb2hex(rgb: string) {
+    const matchedrgb = rgb.match(
+      /^rgba?[\s+]?\([\s+]?(\d+)[\s+]?,[\s+]?(\d+)[\s+]?,[\s+]?(\d+)[\s+]?/i
```

Comment:
It might be helpful to include a comment with an example string that this regex seeks to match, just to make it easier to reason about.